### PR TITLE
Added txtpen in tutorial

### PR DESCRIPTION
**Pull request by rickyhan**
txtpen is the only service that provides highlighting and in-text commenting similar to Medium. I think adding txtpen in the official documentation will benefit new Jekyll users and bring web annotation to more people.

Thanks in advance.
Ricky

### CODE REVIEW

**Comment by rickyhan** [comment]
I have moved the txtpen installation instruction to extras.

Thank you very much :smile:

File: docs/_docs/extras.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ permalink: /docs/extras/
 There are a number of (optional) extra features that Jekyll supports that you
 may want to install, depending on how you plan to use Jekyll.
 
+## Web Highlights and Commenting 
+
+Register your site with [txtpen](https://txtpen.com). Then append 
+
+    <script src="https://txtpen.com/embed.js?site=<your site name>" />
```
Comment (pnn) [inline docs/_docs/extras.md]: could you use fenced code blocks? you can find an example in the `Math Support` section just below this one

Comment (rickyhan) [inline docs/_docs/extras.md]: Done :+1:

**Comment by rickyhan** [comment]
@pnn @pathawks 

Following up on this. What else needs to be done?

**Comment by pathawks** [comment]
@rickyhan I guess the main thing is that we need to decide if this is something that would be useful to a large number of our users. Are there a lot of Jekyll sites using fhis service? Are there resources online to help users integrate the service into their Jekyll site? Are any Jekyll maintainers currently using the service?

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Pull request merged**